Tribler/Core/Upgrade/upgrade.py

```diff
@@ -47,6 +47,7 @@ def run(self):
 
                 self.upgrade_to_tribler7()
                 self.session.config = convert_config_to_tribler71()
+                self.session.config.set_upgrader_enabled(False)
```

Comment:
Line 48/49 should not always be executed and they should only run when necessary (so line 48 should only be executed when upgrading from 6.X -> 7.X and line 49 when upgrading to 7.1).